pkgs/by-name/pa/packet/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cargo,
+  meson,
+  ninja,
+  pkg-config,
+  protobuf,
+  rustPlatform,
+  rustc,
+  wrapGAppsHook4,
+  cairo,
+  dbus,
+  gdk-pixbuf,
+  glib,
+  gtk4,
+  libadwaita,
+  pango,
+  blueprint-compiler,
+  desktop-file-utils,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "packet";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "nozwock";
+    repo = "packet";
+    rev = finalAttrs.version;
+    hash = "sha256-Bk2J38NNrfixkYMveNQb9KkNG/pF3V9LgD1YGgmxg1c=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = "${finalAttrs.src}/Cargo.lock";
+    outputHashes = {
+      "mdns-sd-0.10.4" = "sha256-y8pHtG7JCJvmWCDlWuJWJDbCGOheD4PN+WmOxnakbE4=";
+      "rqs_lib-0.11.5" = "sha256-wD7ZQ5OvIrjNHy3f3ss3bgNJn3l+7cUJjokonCLGE9w=";
+      "sys_metrics-0.2.7" = "sha256-hCialGTBTA7QdiqlTV6sBdiXpgDEB+96IWLFV2M+36o=";
+    };
+  };
+
+  nativeBuildInputs = [
+    cargo
+    meson
+    ninja
+    pkg-config
+    protobuf
+    rustPlatform.cargoSetupHook
+    rustc
+    wrapGAppsHook4
+    blueprint-compiler
+    desktop-file-utils
+  ];
+
+  buildInputs = [
+    cairo
+    dbus
+    gdk-pixbuf
+    glib
+    gtk4
```

Comment:
Nitpick: these should also be in `nativeBuildInputs` since both `glib-compile-resources` and `gtk4-update-icon-cache` are used as command-line tools.

Reply:
sure!

Reply:
Once they're inside `nativeBuildInputs` you don't need to also specify them in `buildInputs` - just once is fine ;p